Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetSnapshotTests.swift

```diff
@@ -483,6 +483,56 @@ class PaymentSheetSnapshotTests: FBSnapshotTestCase {
         verify(paymentSheet.bottomSheetViewController.view!)
     }
 
+    // MARK: Deferred intent tests
+
+    func testPaymentSheet_deferredIntent() {
+        stubNewCustomerResponse()
+
+        let intentConfig = PaymentSheet.IntentConfiguration(mode: .payment(amount: 1000, currency: "USD", setupFutureUsage: .offSession),
+                                                            captureMethod: .automatic)
+
+        preparePaymentSheet(intentConfig: intentConfig)
+        presentPaymentSheet(darkMode: false)
+        verify(paymentSheet.bottomSheetViewController.view!)
+    }
+
+    func testPaymentSheetDarkMode_deferredIntent() {
+        stubNewCustomerResponse()
+
+        let intentConfig = PaymentSheet.IntentConfiguration(mode: .payment(amount: 1000, currency: "USD", setupFutureUsage: .onSession),
+                                                            captureMethod: .automatic)
+
+        preparePaymentSheet(intentConfig: intentConfig)
+        presentPaymentSheet(darkMode: true)
+        verify(paymentSheet.bottomSheetViewController.view!)
+    }
+
+    func testPaymentSheetAppearance_deferredIntent() {
+        stubNewCustomerResponse()
```

Comment:
ooc why test darkmode or appearance + deferred Intent?  They feel unrelated

Reply:
yeah probably not needed, added for good measure but removed them